src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse.json

```diff
@@ -0,0 +1,11 @@
+{
+  "listresourcelimitsresponse": {
+    "resourcelimit": [
+      {
+        "domainid": "%s",
+        "resourcetype": "%s",
+        "max": %s
```

Comment:
nit: missing quotation marks?

Reply:
It is not missing. The "max" is an integer.